lib/CustomFields/components/CustomFieldsForm/tests/CustomFieldsForm-test.js

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import { describe, beforeEach, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import { mount, setupApplication } from '../../../../../tests/helpers';
+
+import CustomFieldsForm from '../CustomFieldsForm';
+import CustomFieldsFormInteractor from './interactor';
+import fetchUsageStatistics from './helpers/fetchUsageStatistics';
+
+describe('CustomFieldsForm', () => {
+  setupApplication();
+
+  const customFieldsForm = new CustomFieldsFormInteractor();
+  const onCancelDelete = sinon.spy();
+  const onConfirmDelete = sinon.spy();
+  const onDeleteClick = sinon.spy();
+  const onFormReset = sinon.spy();
+  const saveCustomFields = sinon.spy();
+  const handleSubmit = sinon.spy();
+
+  const renderComponent = (props) => {
+    return mount(
+      <CustomFieldsForm
+        deleteModalIsDisplayed={false}
+        entityType="user"
+        fetchUsageStatistics={fetchUsageStatistics}
+        fieldsToDelete={[]}
+        initialValues={{
+          customFields: [
+            {
+              id: 'field_1',
+              values: {
+                entityType: 'user',
+                helpText: '',
+                hidden: false,
+                name: 'Facebook ID',
+                order: 1,
+                required: false,
+                type: 'TEXTBOX_SHORT',
+                visible: true,
+              },
+            },
+          ],
+          sectionTitle: 'Custom fields',
+        }}
+        permissions={{
+          canView: true,
+          canEdit: true,
+          canDelete: true,
+        }}
+        viewRoute="/settings/users/custom-fields"
+        onCancelDelete={onCancelDelete}
+        onConfirmDelete={onConfirmDelete}
+        onDeleteClick={onDeleteClick}
+        onFormReset={onFormReset}
+        saveCustomFields={saveCustomFields}
+        onSubmit={handleSubmit}
+        {...props}
+      />
+    );
+  };
+
+  beforeEach(async () => {
+    await renderComponent();
+
+    onCancelDelete.resetHistory();
+    onConfirmDelete.resetHistory();
+    onDeleteClick.resetHistory();
+    onFormReset.resetHistory();
+    saveCustomFields.resetHistory();
+    handleSubmit.resetHistory();
+  });
+
+  it('should display correct section title', () => {
+    expect(customFieldsForm.sectionTitleValue).to.equal('Custom fields');
+  });
+
+  describe('when clicking on an accordion', () => {
+    let initialOpenState;
+
+    beforeEach(async () => {
+      await customFieldsForm.whenLoaded();
+
+      customFieldsForm.customFieldAccordions(0).isOpen;

```

Comment:
is this needed?

Reply:
nope, that's what's left of a `console.log`:)